### ISSUE TITLE
no docker image cp-kafka:4.0.1 available on dockerhub

### ISSUE THREAD

**Issue by missedone** [confluentinc/cp-docker-images#454]
hi folks

I saw the git tag for [v4.0.1|https://github.com/confluentinc/cp-docker-images/releases/tag/v4.0.1], but don't see the docker image cp-kafka:4.0.1 on dockerhub, could you push the image 4.0.1? thanks

**Comment by dylanmei** [comment]
There are images there now, but they appear to contain jars from Kafka 1.0.0 rather than 1.0.1

```
/usr/share/java/kafka# unzip -q -c kafka.jar META-INF/MANIFEST.MF
Manifest-Version: 1.0
Version: 1.0.0-cp1
```

**Comment by zevektor** [comment]
Is there any reason why the connect worker 4.0.1 image shouldn't be using kafka 1.0.1?

**Comment by cricket007** [comment]
@zevektor They are all built from the same base image

**Comment by cricket007** [comment]
@missedone Should this be closed?

**Comment by missedone** [comment]
@cricket007 , well, strictly speaking, I expect to see a the image cp-kafka:4.0.1 has kafka 1.0.1 installed, but what I get is 1.0.0, it makes no sense to me.
----

```
docker run --rm -it confluentinc/cp-kafka:4.0.0 bash -c "ls -l /usr/share/java/kafka/kafka*"
-rw-r--r-- 2 root root 1594891 Nov 21  2017 /usr/share/java/kafka/kafka-clients-1.0.0-cp1.jar
-rw-r--r-- 2 root root   13487 Nov 21  2017 /usr/share/java/kafka/kafka-log4j-appender-1.0.0-cp1.jar
-rw-r--r-- 2 root root  803167 Nov 21  2017 /usr/share/java/kafka/kafka-streams-1.0.0-cp1.jar
-rw-r--r-- 2 root root   46876 Nov 21  2017 /usr/share/java/kafka/kafka-streams-examples-1.0.0-cp1.jar
-rw-r--r-- 2 root root  163050 Nov 21  2017 /usr/share/java/kafka/kafka-tools-1.0.0-cp1.jar
lrwxrwxrwx 1 root root      24 Nov 21  2017 /usr/share/java/kafka/kafka.jar -> kafka_2.11-1.0.0-cp1.jar
-rw-r--r-- 2 root root   55148 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-javadoc.jar
-rw-r--r-- 2 root root    9607 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-scaladoc.jar
-rw-r--r-- 2 root root  806821 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-sources.jar
-rw-r--r-- 2 root root  617406 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-test-sources.jar
-rw-r--r-- 2 root root 5229208 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-test.jar
-rw-r--r-- 2 root root 7452364 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1.jar
```

```
docker run --rm -it confluentinc/cp-kafka:4.0.1 bash -c "ls -l /usr/share/java/kafka/kafka*"
-rw-r--r-- 2 root root 1594891 Nov 21  2017 /usr/share/java/kafka/kafka-clients-1.0.0-cp1.jar
-rw-r--r-- 2 root root   13487 Nov 21  2017 /usr/share/java/kafka/kafka-log4j-appender-1.0.0-cp1.jar
-rw-r--r-- 2 root root  803167 Nov 21  2017 /usr/share/java/kafka/kafka-streams-1.0.0-cp1.jar
-rw-r--r-- 2 root root   46876 Nov 21  2017 /usr/share/java/kafka/kafka-streams-examples-1.0.0-cp1.jar
-rw-r--r-- 2 root root  163050 Nov 21  2017 /usr/share/java/kafka/kafka-tools-1.0.0-cp1.jar
lrwxrwxrwx 1 root root      24 Nov 21  2017 /usr/share/java/kafka/kafka.jar -> kafka_2.11-1.0.0-cp1.jar
-rw-r--r-- 2 root root   55148 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-javadoc.jar
-rw-r--r-- 2 root root    9607 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-scaladoc.jar
-rw-r--r-- 2 root root  806821 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-sources.jar
-rw-r--r-- 2 root root  617406 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-test-sources.jar
-rw-r--r-- 2 root root 5229208 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1-test.jar
-rw-r--r-- 2 root root 7452364 Nov 21  2017 /usr/share/java/kafka/kafka_2.11-1.0.0-cp1.jar
```

we can see both image 4.0.0 and 4.0.1 has exactly same 1.0.0 jars (the size of them are same)

**Comment by cricket007** [comment]
Have you tried 4.0.2 by chance? Should have fixes over 4.0.1 anyway

**Comment by missedone** [comment]
yep, closing this now, 4.0.2 is good.
